docs/source/team.ini

```diff
@@ -0,0 +1,134 @@
+[a]
+# A section for short links
+icl: http://www.imperial.ac.uk
+gh: https://github.com
+
+[dev-institution]
+# Use format
+# institution: institution_website
+#
+# then complete a section with:
+#
+# [institution]
+# department: website
+Imperial College London: http://www.imperial.ac.uk
+University of Oxford: https://www.ox.ac.uk
+Baylor University: https://www.baylor.edu
+University of Washington: https://www.washington.edu
+
+[Imperial College London]
+Department of Mathematics: ${a:icl}/mathematics
+Department of Computing: ${a:icl}/computing
+Department of Earth Science and Engineering: ${a:icl}/ese
+
+[University of Oxford]
+Mathematical Institute: https://www.maths.ox.ac.uk
+
+[Baylor University]
+Department of Mathematics: ${dev-institution:Baylor University}/math
+
+[University of Washington]
+Applied Physics Laboratory: https://www.apl.washington.edu/
+
+[active-team]
+# Use format
+# name : personal-webpage, image-link
+David Ham: http://www.imperial.ac.uk/people/david.ham
+Paul Kelly: http://www.imperial.ac.uk/people/p.kelly
+Lawrence Mitchell:
+Colin Cotter: http://www.imperial.ac.uk/people/colin.cotter
+Rob Kirby: http://www.baylor.edu/math/index.php?id=90540
+Koki Sagiyama: http://www.imperial.ac.uk/people/k.sagiyama
+Nacime Bouziani: https://www.imperial.ac.uk/people/n.bouziani18
+Sophia Vorderwuelbecke: https://www.imperial.ac.uk/people/s.vorderwuelbecke18
+Tom Gregory: https://www.imperial.ac.uk/people/t.gregory18
+Jack Betteridge: https://www.imperial.ac.uk/people/j.betteridge, https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Mallard2.jpg/632px-Mallard2.jpg
+Daniel Shapero: http://psc.apl.uw.edu/people/investigators/daniel-shapero/
+Reuben W. Nixon-Hill: https://www.imperial.ac.uk/people/reuben.nixon-hill10
+Connor Ward: https://www.imperial.ac.uk/people/c.ward20
+Patrick E. Farrell: https://pefarrell.org, https://pefarrell.org/images/profile.jpg
+Pablo Brubeck: , https://pefarrell.org/images/brubeck.jpg
+India Marsden: https://www.maths.ox.ac.uk/people/india.marsden, https://pefarrell.org/images/marsden.jpg
+
+[inactive-team]
+Thomas Gibson:
+Miklós Homolya:
+Tianjiao (TJ) Sun: https://www.doc.ic.ac.uk/~ts2914/
+Andrew McRae: https://www2.physics.ox.ac.uk/contacts/people/mcraea
+Fabio Luporini: http://www.imperial.ac.uk/people/f.luporini12
+Alastair Gregory:
+Michael Lange: https://www.linkedin.com/in/michael-lange-56675994/
+Simon Funke: http://www.simonfunke.com
+Florian Rathgeber: https://kynan.github.io
+Doru Bercea:
+Graham Markall: https://gmarkall.wordpress.com
+
+[contributing-institution]
+Imperial College London: http://www.imperial.ac.uk
+University of Bath: http://www.bath.ac.uk
```

Comment:
What is your obsession with http addresses?

Reply:
These were all copied verbatim from the old documentation, they just need updating. Likewise for team members and contributing users, they are just copied from AUTHORS and the team page

Reply:
Ah, thanks!